package.json

```diff
@@ -144,6 +144,11 @@
       "^.+\\.css$": "<rootDir>/frontend/config/jest/cssTransform.js",
       "^(?!.*\\.(js|jsx|ts|tsx|css|json)$)": "<rootDir>/frontend/config/jest/fileTransform.js"
     },
+    "globals": {
+      "ts-jest": {
+        "diagnostics": false
```

Comment:
what's this for? Can you document the reason for this change in your commit message?